packages/fx-core/src/core/middleware/projectVersionChecker.ts

```diff
@@ -44,21 +46,30 @@ async function showDialog(ctx: CoreHookContext) {
   const lastArg = ctx.arguments[ctx.arguments.length - 1];
   const inputs: Inputs = lastArg === ctx ? ctx.arguments[ctx.arguments.length - 2] : lastArg;
   if (inputs.platform === Platform.VSCode) {
-    await TOOLS?.ui.showMessage(
+    const res = await TOOLS?.ui.showMessage(
       "warn",
-      getLocalizedString("core.projectVersionChecker.vscodeUseNewVersion"),
+      getLocalizedString("core.projectVersionChecker.vscodeUseNewVersion", "Teams Toolkit 5.0.0"),
```

Comment:
Teams Toolkit 5.0 does not exist in marketplace stable release. Should we call it Teams Toolkit 5.0 Pre-Release version?